.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      # Prefix all commit messages with "[docker] "
+      prefix: "[docker] "
+    # Add reviewers
+    reviewers:
+      - "redhat-appstudio/test-team"
```

Comment:
I tried this, it doesn't unpack reviewers into multiple people, but it keep it as alias and send emails to that alias

Reply:
but works for me :D It's just FYI it may work differently than you expect

Reply:
Thanks. Yes, it will send mail the group. I once received the similar mail. So maybe we can use a list of github accounts here.

Reply:
Updated. Thanks@